pkg/util/uint256_test.go

```diff
@@ -12,70 +12,73 @@ import (
 func TestUint256UnmarshalJSON(t *testing.T) {
 	str := "f037308fa0ab18155bccfc08485468c112409ea5064595699e98c545f245f32d"
 	expected, err := Uint256DecodeReverseString(str)
-	if err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, err)
```

Comment:
Same thing here, unrelated changes mixed in the a849a34bc2667a79908a90f5889bf2ba0f0ad6df.